src/bin/wc.rs

```diff
@@ -103,6 +103,10 @@ struct Flags {
     count_bytes: bool,
 }
 
+fn u64_num_digits(val: u64) -> usize {
+    (val as f32).log10() as usize + 1
```

Comment:
Also, https://stackoverflow.com/questions/3272424/compute-fast-log-base-2-ceiling. Just modify it to base 10.